src/Process/FeatureProcessing.js

```diff
@@ -69,6 +69,13 @@ export default {
 
         const extentsDestination = node.getExtentsByProjection(layer.source.projection) || [node.extent];
 
+        const zoomDest = extentsDestination[0].zoom;
+
+        if (zoomDest != layer.zoom.min) {
```

Comment:
Why not having a `zoomDest > layer.zoom.max || zoomDest < layer.zoom.max` too ?

Reply:
because the features are fetched for one tile level (see this [comment](https://github.com/iTowns/itowns/pull/1442#discussion_r459319482))